src/Component.php

```diff
@@ -137,16 +143,28 @@ private function registerAssets()
 
     public function captureMessage($message, $params, $levelOrOptions = [], $stack = false, $vars = null)
     {
+        if (!$this->enabled) {
+            Yii::trace(Json::encode([$message, $params, $levelOrOptions]), __CLASS__);
```

Comment:
Yii::trace is deprecated, please use Yii::debug.

Reply:
All those who use and2.0.14 will not be able to use this extension. I propose to correct this problem, and only then do support 2.0.14